src/scripts/empty-uploads.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Script for database cleanup. Readable, short, works for intended purpose. Keeps us from needing to install the psql cli tool to run a raw sql script.